test/EntityFramework6.Npgsql.Tests/EntityFramework6.Npgsql.Tests.csproj

```diff
@@ -100,5 +101,8 @@
       <Name>EntityFramework6.Npgsql</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
This change is completely unrelated to this feature and should be excluded from the pull request (added by visual studio if you have nunit vs integration I think).